### ISSUE TITLE
'int' object has no attribute 'mean'

### ISSUE THREAD

**Issue by mohamadbahmanpour** [leafinity/keras_ganomaly#4]
Hello!
I am trying to implement ganomaly network on different kind of data but I get error: 'int' object has no attribute 'mean'. problem raises in loss functions when we want to use keras backend as K and take mean over differences. As I know it may happen because of backend either mean is taking an int as input. But I am using same shape for my data as MNIST data to train generator. I don't know what is going wrong. 
Please help me.
Thanks
